README.md

```diff
@@ -28,6 +28,47 @@ through Kafka as more ephemeral and want the flexibility to change how we host K
 In the future we may also apply per-subject permissions to the Avro schemas that
 are stored by the registry.
 
+### Extensions
+
+In addition to the Confluent Schema Registry API, this application provides an
+endpoint that can be used to determine by fingerprint if a schema is already
+registered for a subject.
+
+This endpoint provides a success response that can be cached indefinitely since
+the id for a schema will not change once it is registered for a subject.
+
+`GET /subjects/(string: subject)/fingerprints/(:fingerprint)`
```

Comment:
It's been awhile since I looked at the schema registry API but based on the URL structure I would expect this endpoint to return a "fingerprint" entity. Would it make more sense to add query API to the schemas endpoint e.g. `/schemas?subject=<subject>&fingerprint=<fingerprint>`?